code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -95,8 +95,6 @@
 
 /obj/machinery/media/transmitter/broadcast/attackby(var/obj/item/W, mob/user)
 	. = ..()
-	if(.)
```

Comment:
That's true, it was probably not the best fix, but we were getting weird screwtoggle interactions as a result of it. What side-effects have you noticed?